src/screens/shared/components/PaymentSuccessModal.js

```diff
@@ -42,12 +47,22 @@ class PaymentSuccessModal extends React.PureComponent {
           <p>{`${paymentMethod}`}</p>
         </div>
 
-        <Link
-          to="/investment"
-          className="dropdown-link payment-success-redirect-button"
-        >
-          Back to My Investment
-        </Link>
+        {returnTo === 'My Investment' ? (
+          <Link
+            to="/investment"
+            className="dropdown-link payment-success-redirect-button"
+          >
+            Back to My Investment
+          </Link>
+        ) : (
+          <Link
+            to="/billing"
+            className="dropdown-link payment-success-redirect-button"
+          >
+            Back to Billing
+          </Link>
+        )}
+
```

Comment:
```suggestion
          <Link
            to={returnTo === 'My Investment' ? "/investment" : "/billing"}
            className="dropdown-link payment-success-redirect-button"
          >
            Back to {returnTo}
          </Link>
```

nit: minimize code repetition